examples/transformerconfigs/README.md

```diff
@@ -90,7 +90,7 @@ nameReference:
 ## cusotmizing transformer configurations
 
 Kustomize has a default set of configurations. They can be saved to local directory through `kustomize config save -d`. Kustomize allows modifying those configuration files and using them in `kustomize build` through `-t`.  This tutorial shows how to customize those configurations to
```

Comment:
shouldn't this doc be updated to not mention `-t`?

Reply:
Yep, I'll remove it from the doc. Thank you for catching this.